packages/core/src/components/button/_overrides.scss

```diff
@@ -135,74 +172,117 @@ $pt-gp-dark-icon-color: $g-white;
+        @include custom-button-kind(ghost, $g-dark-primary4, transparent, transparent);
+        @include custom-button-kind(group, $g-dark-grey3, $g-dark-base3);
+      }
     }
 
-    &:disabled,
-    &.#{$ns}-disabled {
-      @include custom-button-kind(primary, $g-dark-primary3, $g-dark-primary4);
-      @include custom-button-kind(secondary, $g-dark-primary5, $g-dark-base3);
-      @include custom-button-kind(stroked, $g-dark-grey4, transparent, $g-dark-grey4);
-      @include custom-button-kind(ghost, $g-dark-primary4, transparent, transparent);
-      @include custom-button-kind(group, $g-dark-grey3, $g-dark-base3);
+    &.#{$ns}-intent-danger {
+      @include custom-button-kind(primary, $g-white, $g-error-600);
+      @include custom-button-kind(secondary, $g-error-600, $g-dark-base4);
+      @include custom-button-kind(stroked, $g-dark-grey1, transparent, $g-error-600);
+      @include custom-button-kind(ghost, $g-error-600, transparent, transparent);
+
+      &:hover {
+        @include custom-button-kind(primary, $g-white, $g-error-500);
+        @include custom-button-kind(secondary, $g-error-500, $g-dark-base5);
+        @include custom-button-kind(stroked, $g-dark-grey1, $g-dark-base5, $g-error-500);
+        @include custom-button-kind(ghost, $g-error-500, transparent, transparent);
+      }
+
+      &:active,
+      &.#{$ns}-active {
+        @include custom-button-kind(primary, $g-white, $g-error-600, $g-error-700);
+        @include custom-button-kind(secondary, $g-error-700, $g-dark-base4);
+        @include custom-button-kind(stroked, $g-white, transparent, $g-error-800);
+        @include custom-button-kind(ghost, $g-error-700, transparent, transparent);
+      }
+
+      &:disabled,
+      &.#{$ns}-disabled {
+        @include custom-button-kind(primary, $g-error-800, $g-error-600);
+        @include custom-button-kind(secondary, $g-error-800, $g-dark-base3);
+        @include custom-button-kind(stroked, $g-dark-grey4, transparent, $g-error-800);
+        @include custom-button-kind(ghost, $g-error-800, transparent, transparent);
+      }
     }
 
   }
 }
 
 // CUSTOM class for extra-small and extra-large sizes:
 .#{$ns}-button {
-  &.#{$ns}-g-extra-small {
+  &.#{$ns}-large {
+    min-height: 40px;
+    min-width: 40px;
+    padding-left: 15px;
+    padding-right: 15px;
+    border-radius: 20px;
+  }
+
+  &.#{$ns}-g-extra-extra-small {
     min-height: 16px;
     min-width: 16px;
     padding-left: 0px;
     padding-right: 0px;
     padding-top: 0px;
     padding-bottom: 0px;
+
     .#{$ns}-icon>svg {
       height: 10px;
       width: 10px;
     }
   }
 
-  &.#{$ns}-g-extra-large {
-    min-height: 40px;
-    min-width: 40px;
-    padding-left: 15px;
-    padding-right: 15px;
-    border-radius: 20px;
-  }
+  &.#{$ns}-g-extra-small {
+    min-height: 20px;
+    min-width: 20px;
+    padding-left: 4px;
+    padding-right: 4px;
+    padding-top: 0px;
+    padding-bottom: 0px;
```

Comment:
The same here, this can be shortened to:  `padding: 0 4px;` I'm not a superfan of using shortcuts (because CSS have a lot of them with a lot of variations 😆 ) but this one is very common.